src/components/footer.vue

```diff
@@ -1,93 +1,111 @@
-          </div>
-
-          <div class="col-md-2 mb-3">
-            <h6 class="text-uppercase font-weight-bold">
-              <a href="#!">Awards</a>
-            </h6>
-          </div>
-
-          <div class="col-md-2 mb-3">
-            <h6 class="text-uppercase font-weight-bold">
-              <a href="#!">Help</a>
-            </h6>
-          </div>
-
-          <div class="col-md-2 mb-3">
-            <h6 class="text-uppercase font-weight-bold">
-              <a href="#!">Contact</a>
-            </h6>
-          </div>          
-        </div><!--row text-center d-flex justify-content-center pt-5 mb-3-->
-
-    <hr class="rgba-white-light" style="margin: 0 15%;">
-    <div class="row d-flex text-center justify-content-center mb-md-0 mb-4">
-      <div class="col-md-8 col-12 mt-5">
-        <p style="line-height: 1.7rem">Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur.</p>
-      </div>
-    </div>    
-    <hr class="clearfix d-md-none rgba-white-light" style="margin: 10% 15% 5%;">
-
-
-    <div class="row pb-3">
-      <div class="col-md-12">
-
-        <div class="mb-5 flex-center">
-            <a class="fb-ic">
-              <i class="fa fa-facebook fa-lg white-text mr-md-4"> </i>
-            </a>
-            <a class="tw-ic">
-              <i class="fa fa-twitter fa-lg white-text mr-md-4"> </i>
-            </a>
-            <a class="gplus-ic">
-              <i class="fa fa-google-plus fa-lg white-text mr-md-4"> </i>
-            </a>
-            <a class="li-ic">
-              <i class="fa fa-linkedin fa-lg white-text mr-md-4"> </i>
-            </a>
-            <a class="ins-ic">
-              <i class="fa fa-instagram fa-lg white-text mr-md-4"> </i>
-            </a>
-            <a class="pin-ic">
-              <i class="fa fa-pinterest fa-lg white-text"> </i>
-            </a>
+<template>                     
+    <div class="footer-wrap">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-4 col-sm-4">
+                    <h3 class="text-center">ABOUT Your Housing</h3>
+                    <div class="footer-logo text-center"><img src="../assets/housingICON.png" style="height: 100px;"></div>
+                    <p>YourHousing is a project started by young uprising proffesionals from The Netherlands. At the time of project creations all team members were still students.
+                    Our main goal is to provide best possible freemium solution without 3rd parties for accommodation in The Netherlands. </p>
+                </div>
+                <div class="col-md-2 col-sm-2 text-center">
+                    <h3>Quick  LInks</h3>
+                    <ul class="footer-links text-center">
```

Comment:
<router-link to=""> is shortcut for <a href=""> so when it will be rendered it will be <a href="#."><a href="/>Home</a></a>